python/lsst/ts/salobj/base_csc.py

```diff
@@ -183,7 +177,15 @@ def __init__(
                     f"simulation_mode={simulation_mode} "
                     f"not in valid_simulation_modes={self.valid_simulation_modes}"
                 )
-        self.evt_simulationMode.set(mode=int(simulation_mode))
+
+        self._requested_simulation_mode = int(simulation_mode)
+        self._settings_to_apply = settings_to_apply
+        self._summary_state = State(self.default_initial_state)
+        self._initial_state = initial_state
+        self._faulting = False
+        self._heartbeat_task = base.make_done_future()
+        # Interval between heartbeat events (sec)
+        self.heartbeat_interval = HEARTBEAT_INTERVAL
 
         if not hasattr(self, "version"):
```

Comment:
I noticed that this class variable is not documented in the class attributes section.

Reply:
Good catch. Added.